### PR TITLE
i18n: add zh-CN localization

### PR DESCRIPTION
**Pull request by fakelbst**

### CODE REVIEW

File: src/i18n/zh-CN.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+(function(jsGrid) {
+
+    jsGrid.locales.es = {
```
Comment (tabalinas) [inline src/i18n/zh-CN.js]: could you put `jsgrid.locales["zh-cn"] = {`?

**Comment by tabalinas** [comment]
Thank you for the PR. Could you do a small correction from my comment?

**Comment by fakelbst** [comment]
I'm sorry about my mistake.I just fixed it.😬

**Comment by tabalinas** [comment]
Thank you very much!

**Pull request merged**